### PR TITLE
Trevor Suter IC wk 4

### PR DESCRIPTION
**Pull request by Trevorsuter**

### CODE REVIEW

File: lib/museum.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  def add_exhibit(exhibit)
+    exhibits << exhibit
+  end
+
+  def recommend_exhibits(patron)
+    exhibits.find_all do |exhibit|
+      patron.interests
+    end
+  end
```
Comment (mikedao) [inline lib/museum.rb:18]: This isn't doing what you expect it to, it is returning all of the exhibits. This would have been caught with some tighter testing.

File: lib/museum.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      end
+    end
+    pbei
+  end
+
+  def ticket_lottery_contestants(exhibit)
+    patrons.find_all do |patron|
+      if patron.spending_money <= exhibit.cost
+        patron
+      end
```
Comment (mikedao) [inline lib/museum.rb:46]: We don't need this conditional here, check the documentation for this enumerable.

**Review by mikedao — COMMENTED** [review]
When you're writing your tests, try to cover more edge cases to cover different situations. They aren't verifying that your methods are really doing what you want them to be doing.

Score: 2
